client/src/components/MainPage/MainCategoryList/style.scss

```
@@ -0,0 +1,29 @@
+@import 'src/styles/package';
+
+.main-category-list {
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  margin: 0 auto;
+  padding: 10px;
+  .category-text-wrap {
+    display: flex;
+    flex-direction: row;
+    padding: 10px;
+    .category-time-text {
+      font-size: 1rem;
+      color: #3c3c3c;
```

scope가 달라지는 부분에서 한줄 띄줘주기를 하면 가독성이 더 증가합니다!
다음에는 한번 띄워보시는 걸 추천드려요!!

```suggestion
  flex-direction: column;
  margin: 0 auto;
  padding: 10px;
  
  .category-text-wrap {
    display: flex;
    flex-direction: row;
    padding: 10px;
    
    .category-time-text {
      font-size: 1rem;
      color: #3c3c3c;
```